### ISSUE TITLE
Language specific URL alias patterns missing in content type configuration

### ISSUE THREAD

**Issue by olafgrabienski** [backdrop/backdrop-issues#6405]
## Description of the bug

On a multilingual site with content translation it's easy to miss the option to provide language specific URL alias patterns.

## Steps To Reproduce

To reproduce the behavior:
1. Go to the modules page, and enable "Content Translation".
2. Add a language other than English to your Backdrop site, e.g. Spanish.
3. Configure a content type, enable Multilingual support (with or without translation), and save the content type.
4. Edit the content type again, and have a look at the URL alias pattern settings.

## Actual behavior

There is only one field for the "Default URL alias pattern".

## Expected behavior

The settings should include fields for language specific patterns (language neutral, English and Spanish in this case).

Alternatively, there should be a link to the general URL alias patterns page (`admin/config/urls/path/patterns`) where all language specific options are available.

## Additional information

Add any other information that could help, such as:
- Backdrop CMS 1.27.0

**Comment by olafgrabienski** [comment]
I _love_ the option to configure URL aliases in the content type configuration (introduced in https://github.com/backdrop/backdrop-issues/issues/899). It's so useful that I didn't visit the general URL alias patterns for a long time. This way, I literally _forgot_ the option for language specific URL alias patterns. These options can be very useful, though, so let's not hide them too far away.

**Comment by olafgrabienski** [comment]
Not sure which approach is better: include fields for language specific patterns in the vertical tab, or link to the general URL alias patterns page? See mockups for a comparison:

**(a) Multilingual patterns in vertical tab**

![backdrop-page-config-url-patterns-form](https://github.com/backdrop/backdrop-issues/assets/14188307/114d1077-9127-482d-81ac-aa524667c748)

**(b) Link to general URL alias patterns page**

![backdrop-page-config-url-patterns-link](https://github.com/backdrop/backdrop-issues/assets/14188307/762ff0e5-923f-411b-9810-7bc517118884)

**Comment by kiamlaluno** [comment]
I would rather give a link to the page to set multilingual patterns.

**Comment by herbdool** [comment]
I don't know which is better UI. The first option helps with the workflow. People don't have to find their way back to the content type after setting the patterns.

**Comment by olafgrabienski** [comment]
Thanks for your feedback, @kiamlaluno and @herbdool.

Before creating the mockups, I was skeptical if the first option (form in vertical tab) would be too busy, but in the meantime I like it. It's indeed good when people don't have to go somewhere _and_ return before they continue to configure the content type. Also, I think, seeing the form in the tab it's easier to understand what multilingual patterns are.

> I would avoid the first proposed solution just to avoid to duplicate a form on a different page.

I don't know if there is a technical impact when a form is duplicated. That said, we are already duplicating the "Default URL alias pattern", and wasn't that the point of #899?

**Comment by kiamlaluno** [comment]
I was thinking of the issue we had because the same form in different pages shown different permissions selected, despite the selected permissions should have been the same.  
The duplicated form in this case is probably simpler; the chances to have a similar issue are lower.

**Comment by kiamlaluno** [comment]
(In my previous comment, I am referring to #6117.)

**Comment by yorkshire-pudding** [comment]
I think placing the options on the content type form is better as they don't have to leave the form; this mirrors what we do for permissions.

One question, and this may be my ignorance of multilingual sites, is there a difference between "Default URL alias pattern" and "Language neutral"?

**Comment by olafgrabienski** [comment]
> is there a difference between "Default URL alias pattern" and "Language neutral"?

Yes, there is a difference. Home my explanation below is helpful.

- "Language neutral" is a general state of multilingual content and not necessarily related to URL alias patterns. Content gets this state when you create a content item without selecting a language. This is also the case for content which was created before a content type was configured to allow multilingual content.
- The "Default URL alias pattern" is a fallback for not specified patterns. Example: If the pattern for English is not specified, the Default pattern is used.
